pkg/js-search/source/IndexStrategy/ExactWordIndexStrategy.test.ts

```diff
@@ -0,0 +1,16 @@
+import { ExactWordIndexStrategy } from "./ExactWordIndexStrategy";
+describe('ExactWordIndexStrategy', function () {
+  var indexStrategy;
+  beforeEach(function () {
```

Comment:
Nitpicky with the dose of `var` alergies:

I would replace all occurrences of `var` with `const` (probably using `find` + `sed`), run `tsc` to see if it complains about reassigning to a `const` variable, at which point, I would replace all error occurrences with `let`.

Furthermore, for future readability I would replace all callbacks with `() => { ... }` rather than `function () { ... }`.
I believe this could partly be done with `sed` as well...or even the eslint set up to prefer arrow function callbacks and run `eslint --fix` (I will look more into it)

Finally, this is a preference when writing new code, and since, I understand, this has been generated using some flow->TS transpiler, it's understandable if you've used the "If it works don't touch it" logic (and it most certainly does)

Reply:
The codebase is now allergen-free! Arrow functions are now preferred over `function () {}`.